### DISCUSSION TITLE
Blockquote Author or Source

### DISCUSSION THREAD

**Discussion by airsiena** [McShelby/hugo-theme-relearn#443] [Q&A] [answered]
At the top of the page [https://mcshelby.github.io/hugo-theme-relearn/cont/markdown/](https://mcshelby.github.io/hugo-theme-relearn/cont/markdown/) there is a citation where the author appears in the lower right corner.
Can this be achieved by formatting the content only with Markdown `> ` or do I have to use the HTML `<cite>` tag and style it by hand?

**Comment by McShelby** [comment, marked as answer]
This is currently not natively possible with markdown only.

Besides writing a shortcode, some guys implemented this by using code fences as explained in [this post](https://discourse.gohugo.io/t/blockquote-with-render-hook/40701).

Nevertheless I currently don't plan to support this.
